### DISCUSSION TITLE
running UFS-P8 with no aerosol and no wave

### DISCUSSION THREAD

**Discussion by chulsushin** [ufs-community/ufs-weather-model#2169] [Q&A] [answered]
I have been running UFS-P8 C384 on TACC Frontera and now I'm trying to run the same UFS-P8 but with no aerosol and no wave. What I have found so far is that I need to:

1) recompile by "cmake -DAPP=S2S -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 -DINLINE_POST=ON" (Note that I used "S2S" instead of "S2SWA" here.)

2) modify "nems.configure" based on "news.configure.cpld_noaero_nowave.IN"
 
Could anyone let me know if those steps are correct and there is anything else I should change? Any comments are welcome.

Thanks,
Chul-Su

**Comment by gspetro-NOAA** [comment, marked as answer]
Hi Chul-Su,

That looks like a good start! You will also want to be sure that you have a compatible version of the stack, but if you have already been running p8 on TACC Frontera, it sounds like you may be set. @jkbk2004 @ulmononian Anything to add? 

Best,
Gillian Petro

**Reply by chulsushin** [reply]
Thanks, @gspetro-NOAA!
